coforma/swift-tech-challenge
# Populate cards from institutions data

# Summary

- Add an `institutions` utility that makes fetching and parsing the data easier on the components
- Add tests
- Move dynamodb-libs to libs/ folders






Fixes #24 

## Type of change


# How To Test
- Export AWS credentials in your terminal
- Run `yarn dev`
- Load `localhost:3000/`
- Verify it shows a handful of cards with data from the database

# Checklist:
